<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewCustOrder</class>
 <widget class="QMainWindow" name="ViewCustOrder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Enter Customer SSN</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="SSN_textedit">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item alignment="Qt::AlignRight">
     <widget class="QPushButton" name="Enter_SSN_btn">
      <property name="text">
       <string>Enter</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Orders made by a customer</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="Orders_listWidget"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="Select_Order_btn">
          <property name="text">
           <string>Select</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Products in Selected Order</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="OrderProducts_listWidget"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item alignment="Qt::AlignLeft">
     <widget class="QPushButton" name="back_btn">
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
